lib/renderer/renderer-webpack.js

```diff
@@ -91,30 +112,30 @@ class Renderer {
         this.vue = options.vue ?
             {app: options.vue.app, server: options.vue.server, client: options.vue.client} :
             {app: undefined, server: undefined, client: undefined};
-        this.bundlePath = path.join(this.rootPath, ".expressvue", "bundles");
+        this.expressVueFolder = path.join(this.rootPath, ".expressvue");
     }
     /**
      * @param {boolean} [forceBuild=false]
      * @returns {Promise<boolean>}
      */
     async Bootstrap(forceBuild = false) {
         const vm = this;
-        await mkdirp.sync(vm.bundlePath);
+        await mkdirp.sync(vm.expressVueFolder);
```

Comment:
`await` should not be there since you are using `mkdirp.sync`

Reply:
good catch, also the recasting of this to vm is unneded too..